nemo/lightning/pytorch/callbacks/__init__.py

```diff
@@ -15,6 +15,7 @@
 from nemo.lightning.pytorch.callbacks.ddp_parity_checker import DdpParityChecker
 from nemo.lightning.pytorch.callbacks.debugging import ParameterDebugger
 from nemo.lightning.pytorch.callbacks.garbage_collection import GarbageCollectionCallback
+from nemo.lightning.pytorch.callbacks.jit_transform import JitTransform
```

Comment:
I am not sure if we add the model accelerators as callbacks into the Nemo. Let's chat.